speedy-recovery-client/src/components/secured/conversation/ConversationPage.js

```diff
@@ -27,6 +27,7 @@ import { getSenderMessageNum, messageMap, setupMessages } from "../../../service
 import "./ConversationPage.css";
 
 class ConversationPage extends Component {
+  _isMounted = false;
```

Comment:
So are we doing it this way then even though Fabihas Link showed that the cleaner approach would be to have a `canceablePromise`?

Reply:
I tried the CancellablePromise, which needs to either use class CancellablePromise<T> extends Promise<T> or implement it following this makecancelable link: https://www.npmjs.com/package/makecancelable
Both of them needs to refactor, I'm afraid it will break current functionality, just don't think it's worthy to refactor for this warning. 
Instead, to change the timer to 5 seconds could stop the warning. Do you think we should just change the timer to a longer time?

Reply:
Yeah let's leave it as it is, gvien that we are already 1 day late.